more_itertools/more.py

```diff
@@ -1004,3 +1004,53 @@ def divide(n, iterable):
         ret.append(iter(seq[start:stop]))
 
     return ret
+
+
+def always_iterable(obj):
+    """
+    Given an object, always return an iterable.
+
+    If the object is not already iterable, return a tuple containing containing
+    the object::
+
+        >>> always_iterable(1)
+        (1,)
+
+    If the object is ``None``, return an empty iterable::
```

Comment:
I almost removed this special case in order to simplify the docstring... "Given an object, always return an iterable. If the object is not already iterable, wrap it in a tuple and return it. Otherwise, return the object."

I could argue both ways for whether `None` should return `(None,)` or whether it should return `()`. Anybody feel strongly?